CHANGELOG.md

```diff
@@ -28,6 +28,10 @@ These changes are available on the `master` branch, but have not yet been releas
   ([#2390](https://github.com/Pycord-Development/pycord/pull/2390))
 - Fixed `NameError` in some instances of `Interaction`.
   ([#2402](https://github.com/Pycord-Development/pycord/pull/2402))
+- Fixed `PartialMessage/able` IDs sometimes being `str` rather than `int`.
+  ([#2406](https://github.com/Pycord-Development/pycord/pull/2406))
+- Fixed Views sent in `ForumChannel.create_thread` not recieving interactions.
+  ([#2406](https://github.com/Pycord-Development/pycord/pull/2406))
```

Comment:
```suggestion
```
Why is that in here

Reply:
I feel like it's duplicate

Reply:
This was the main issue i found which led to me making this fix, but if it's not necessary then fair enough

Reply:
I mean the real change is the first entry. Maybe combine it?